frontend/src/scenes/insights/Insights.js

```diff
@@ -224,10 +239,10 @@ function _Insights() {
                                 <Card
                                     title={
                                         <div className="float-right">
-                                            {showIntervalFilter[activeView] && (
+                                            {showIntervalFilter(allFilters) && (
```

Comment:
Could be irrelevant as I can't reproduce it, but when i first loaded this PR, it seemed to lag before the filters were shown which could have to do with the filter logic depending on `allfilters` now